src/init/autoApplyCustomizations.ts

```diff
@@ -6,7 +6,7 @@ export function manageAutoApplyCustomizations(
   userConfig: IVSIcons,
   applyCustomizationCommand: () => void): void {
   if (!isNewVersion) { return; }
-  const propObj = packageJson.contributes.configuration.properties || {};
+  const propObj: any = packageJson.contributes.configuration.properties as object;
```

Comment:
It's curious because I'm also using 2.2.1 and I'm not seeing this warning. My previous comment was more about why you annotate the variable `propObj` with two types if, in the end, typescript is going to use the `any` (in this case). If you want to use `object` why the `any`? And if you want to use `any`, why the `object`?